src/components/chat/chat.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
```

Comment:
This component should be inside `containers` folder because it is connected to the redux. Also, it is better to move `ActiveMessenger` inside the folder of `chat` component because it is going to be used only there, in `chat`

Reply:
now component active-messenger inside folder chat, and component chat inside containers